### PR TITLE
Merged '29 pause button pop up' branch to development

### LINKED ISSUE #29: pause button (pop up)

### PR DESCRIPTION
**Pull request by jiroblea**
Developed:
- working pause button
- UI of pause popup
- working buttons of pause popup

Please see the image:
image

### CODE REVIEW

File: iskrambol/playscreen.kv
Comment (Serv08) [inline iskrambol/playscreen.kv:1]: back button in Settings from Pause window returns to main menu screen instead of Pause window

**Review by Serv08 — CHANGES_REQUESTED** [review]
Fix settings' back button from pause window

**Comment by jiroblea** [comment]
> Fix settings' back button from pause window

That should be possible. We should open another issue for that.

**Comment by Serv08** [comment]
> > Fix settings' back button from pause window
> 
> That should be possible. We should open another issue for that.

ok. Let's stick with that

**Review by Serv08 — APPROVED** [review]

**Pull request merged**